### PR TITLE
Change stack analysis endpoint in tests

### PR DESCRIPTION
**Pull request by tisnik**

### CODE REVIEW

File: integration-tests/features/steps/common.py [hunk 1]
```diff
@@ -325,8 +325,8 @@ def send_manifest_to_stack_analysis(context, manifest, name, endpoint, use_token
 
 
 def stack_analysis_endpoint(context, version):
-    endpoint = {"1": "/api/v1/stack-analyses/",
-                "2": "/api/v1/stack-analyses-v2/"}.get(version)
+    endpoint = {"1": "/api/v1/stack-analyses-v1/",
```
Comment (samuzzal-choudhury) [inline integration-tests/features/steps/common.py]: @tisnik unfortunately, we won't have any endpoint /stack-analyses-v1. There is one and only one endpoint /stack-analyses.

Comment (tisnik) [reply integration-tests/features/steps/common.py]: oh ok, thanks for info. I just followed the msg 'changing /stack-analyses-v2 to /stack-analyses and /stack-analyses to /stack-analyses-v1'. Anyway the "1" version is not used in any test AFAIK.

**Comment by tisnik** [comment]
@samuzzal-choudhury could you please review the proposed changes? Now it is not mandatory to use the 'version 2' in test steps.

**Comment by tisnik** [comment]
@samuzzal-choudhury WDYT?

**Pull request merged**